dbt_airflow_factory/k8s/k8s_parameters.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         envs: Optional[Dict[str, str]] = None,
         secrets: Optional[List[Secret]] = None,
         is_delete_operator_pod: bool = True,
+        config_file: Optional[str] = None,
```

Comment:
In the documentation, I can see that the default value is "~/.kube/config" so maybe it would be better to provide the same to not overwrite it with None?

Reply:
Changed!